### PR TITLE
Fixes non-login user editing

### PR DESCRIPTION
**Pull request by johnclary**
## Associated issues
- https://github.com/cityofaustin/atd-data-tech/issues/13153'

This also resolves a bug i created [here](https://github.com/cityofaustin/atd-moped/pull/1085/files#diff-0aaf9775780004fb06632bc543b836ac98b0f690c74ec3f8862609e75749094e) which broke the users query. Sorry I missed this 😑 

## Testing
**URL to test:**  [Netlify](https://deploy-preview-1087--atd-moped-main.netlify.app/) or local

**Steps to test:**
1. Go to the staff page and create a non-login user
2. Edit that user by changing the name or workgroup
3. ☝️ please do not edit any users you did not create — the staging user pool needs to stay in sync with the access DB for the migration.

---
#### Ship list
- [ ] Code reviewed 
- [ ] Product manager approved
- [ ] Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#) if applicable

### CODE REVIEW

File: moped-editor/src/views/staff/StaffForm.js [hunk 1]
```diff
@@ -257,6 +257,7 @@ const StaffForm = ({
                 render={({ field }) => (
                   <Select
                     {...field}
+                    value={field.value || ""}
```
Comment (johnclary) [inline moped-editor/src/views/staff/StaffForm.js:260]: squashes an MUI linter warning

File: moped-editor/src/queries/staff.js [hunk 2]
```diff
@@ -25,7 +25,6 @@ export const GET_USER = gql`
       first_name
       is_coa_staff
       last_name
-      staff_uuid
```
Comment (mddilley) [inline moped-editor/src/queries/staff.js:28]: I missed this too! Reminds me of your dev sync share yesterday about the challenge of keeping DB schema/backend/frontend all in sync

**Review by mddilley — APPROVED** [review]
Thanks for fixing this! I tested locally, and it works perfectly 🚀

File: moped-editor/src/views/projects/projectView/ProjectWorkActivitiesTable.js [hunk 3]
```diff
@@ -154,7 +154,7 @@ const ProjectWorkActivitiesTable = () => {
                   startIcon={<AddCircleIcon />}
                   onClick={props.action.onClick}
                 >
-                  Add Contract
+                  Add Work
```
Comment (johnclary) [inline moped-editor/src/views/projects/projectView/ProjectWorkActivitiesTable.js:157]: just going to sneak this in here rather than open another PR 🙈

**Review by chiaberry — APPROVED** [review]
✔️

**Pull request merged**